### DISCUSSION TITLE
Seconds and calendar week, only

### DISCUSSION THREAD

**Discussion by PescheHelfer** [martinet101/ElevenClock#529] [General]
ElevenClock is almost perfect now, but I have the issue that it covers the battery status, which is quite important on a laptop. The only things I am missing in the system clock are the seconds and the calendar week. Would it be possible to add an option to only show this information on one line? e.g. W04 :55   or W04|55
That way I might be able to squeeze this single line above the system clock, while still being in the taskbar region without hiding anything. Currently, I have moved ElevenClock to be above the regular clock, but to see seconds and the calendar week, you need two lines, so it does not fit in the taskbar together with the regular clock.
Thanks for considering this and thanks for all the great work!

Edit: Shifting the clock more to the right reveals the battery symbol, while showing the full clock. But it a very tight fit! It would also help if we could a shorter version (option) for the calendar week. i.e. instead of (W04) 29.01.22  --> 29.01.22 W04 or 04 | 29.01.22

**Comment by marticliment** [comment]
For the moment, you can set a smaller font size. 

Note that this feature will eventually come (#380)
